bdb/file.c

```diff
@@ -7853,9 +7853,18 @@ int bdb_check_files_on_disk(bdb_state_type *bdb_state, const char *tblname,
     assert(bdb_state->parent == NULL);
 
     if (bdb_state->attr->keep_referenced_files) {
-        lognum = bdb_get_last_logfile(bdb_state, bdberr);
-        if (lognum == -1)
-            return -1;
+        int ourlowfilenum;
```

Comment:
If this is the same logic as log deletion, want to put it in a function and call it from all 3 places?

Reply:
done